src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,8 +12,10 @@
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_INDEX = new Prefix("-i");
-    public static final Prefix PREFIX_ROOM = new Prefix("-r");
     public static final Prefix PREFIX_DATE_START = new Prefix("from/");
     public static final Prefix PREFIX_DATE_END = new Prefix("to/");
-    public static final Prefix PREFIX_GUEST = new Prefix("-g");
+    public static final Prefix PREFIX_ROOM = new Prefix("r/");
+    public static final Prefix PREFIX_GUEST = new Prefix("g/");
```

Comment:
`PREFIX_GUEST` seems to not be in use. If it's for identifying a `Guest`, the current AB4 structure supports identifying by the index. In this case then, `PREFIX_INDEX` would be more suitable.

Reply:
More suitable for?

Reply:
`PREFIX_INDEX` would be more suitable for identifying a `Guest` from the displayed list than `PREFIX_GUEST`.

Is there a use for `PREFIX_GUEST`?